torchci/clickhouse_queries/query_execution_metrics_individual/query.sql

```diff
@@ -8,9 +8,15 @@ SELECT
     quantile(0.5)(memory_usage) as memoryBytesP50,
     count(*) as num
 FROM
-    clusterAllReplicas(default, system.query_log)
+    clusterAllReplicas(default, default.all_query_logs)
 where
-    event_time >= {startTime: DateTime64(3)}
+    -- for partitioned tables
+    toYYYYMM(event_date) >= toYYYYMM({startTime: DateTime64(3)})
+    and toYYYYMM(event_date) <= toYYYYMM({stopTime: DateTime64(3)})
```

Comment:
Curious: why is this necessary?  event_date has filters below, does it just not realize it can generalize from the filters later?